src/content/docs/new-relic-solutions/new-relic-solutions/plan-your-cloud-adoption/identify-issues-roadblocks.mdx

```diff
@@ -164,8 +164,4 @@ SELECT count(*) FROM JavaScriptError WHERE appName = '<<var>BrowserAppName</var>
 * Replace `<ErrorClass1>` and `<ErrorClass2>` with the on-premise error class names that you do **not** want New Relic to alert you about.
 * Set the [threshold](/docs/alerts/new-relic-alerts/defining-conditions/define-thresholds-trigger-alert) based on your alerting needs.
 
-Using this query, New Relic alerts you every time a JavaScript error with an error class that is not normally reported occurs.
-
-## For more help [#more_help]
-
-Explore the [New Relic Platform](https://newrelic.com/products).
+Using this query, New Relic alerts you every time a JavaScript error with an error class that is not normally reported occurs.
```

Comment:
the verb at the end makes this sentence a bit awkward but I suspect you were not focusing on that in this sweep :-) But might read better as "Using this query, New Relic alerts you every time a JavaScript error occurs when it has an error class that is not normally reported."

Reply:
👍